### DISCUSSION TITLE
Repeatable fields prefilled

### DISCUSSION THREAD

**Discussion by Mte90** [Laravel-Backpack/community-forum#1215] [Q&A (Help)] [answered]
So I have a repeatable field that it will be populated by the user but now asked to get the various items aalready prefilled so he just need to configure some values and not everything.
Looking at the doc there isn't a way to define what should be the various items already available.

Maybe a solution can be something like in `setupCreateOperation` and update too:

```
CRUD::field('repeatablefield')->value('<json skeleton off all the items with their subfields and placeholder value>');
```

[a comment by an automated account is not shown]

**Reply by Mte90** [reply]
Useful but doens't allow to specify some fields in the repeatable as disabled based on the input...

**Reply by Mte90** [reply]
With a lot of code I was able to do it, also I discovered that if a field is disabled the value isn't saved, switched to readonly.

**Reply by tabacitu** [reply]
Yup @Mte90 - 100%, that's the difference between `disabled` and `readonly`. It's an HTML thing, not a Backpack thing:
- `disabled` means it doesn't pass the value to the HTML form;
- `readonly` means it will pass the value, but the user can't edit it;

If you ask me, the naming should be exactly the opposite in the HTML spec... but hey... it is what it is.

--

If you can come back with the code that made it work for you, I'm sure it will help someone someday. And even if not, it might help our AI learn from it, so next time you ask, it'll know 😉
